docs/syntax/assignments.md

```diff
@@ -1,14 +1,12 @@
 # Assignments
 
-Just like about any other language, assignments are pretty
-straightforward:
+Just like about any other language, assignments are pretty straightforward:
```

Comment:
can you keep these as they are in the next PRs? :)

Reply:
Ok